src/test/java/teammates/test/cases/action/AdminEmailLogPageActionTest.java

```diff
@@ -95,12 +98,12 @@ public void removeAndRestoreLogMessage() {
         gaeSimulation.loginAsAdmin("admin");
         gaeSimulation.clearLogs();
 
-        Date twoDaysAgo = TimeHelper.getDateOffsetToCurrentTime(-2);
-        insertLogMessagesAtTime(logMessages.get(LOG_MESSAGE_INDEX_TWO_DAYS_AGO), twoDaysAgo.getTime());
-        Date yesterday = TimeHelper.getDateOffsetToCurrentTime(-1);
-        insertLogMessagesAtTime(logMessages.get(LOG_MESSAGE_INDEX_YESTERDAY), yesterday.getTime());
-        Date today = TimeHelper.getDateOffsetToCurrentTime(0);
-        insertLogMessagesAtTime(logMessages.get(LOG_MESSAGE_INDEX_TODAY), today.getTime());
+        Instant twoDaysAgo = TimeHelper.getInstantDaysOffsetFromNow(-2);
```

Comment:
A lot of Instants used in this test class are common, e.g. `twoDaysAgo`, `yesterday`, `today`. Is it wise to extract them to static variables in the test class and reuse them?